pkg/aws/services/vpclattice.go

```diff
@@ -13,6 +14,12 @@ import (
 
 //go:generate mockgen -destination vpclattice_mocks.go -package services github.com/aws/aws-application-networking-k8s/pkg/aws/services Lattice
 
+type Tags = map[string]*string
```

Comment:
re-declaration from cloud.go, we can remove it then from cloud.go

Reply:
Done